aspnetcore/tutorials/index.md

```diff
@@ -16,6 +16,7 @@ ms.prod: asp.net-core
 The following step-by-step guides for developing ASP.NET Core applications are available:
 
```

Comment:
The image at https://github.com/aspnet/Docs/pull/3090/files#diff-9bd6e4ce4447edef6d1d5cb66d28aca6 still has port 61493; not critical, just FYI.  (Can't put a comment on an image so putting it in the next file in the diff)